packages/11ty/_layouts/cover.liquid

```diff
@@ -11,7 +11,8 @@ layout: base.11ty.js
   <div
     class="quire-cover__overlay"
     style="background-image: url('{{ imagePath }}');"
-  />
+  >
+  </div>
```

Comment:
this self-closing tag was also inserting a bunch of unnecessary HTML space entities in the EPUB output